src/bundle/Resources/translations/forms.en.xliff

```diff
@@ -421,6 +421,66 @@
         <target>State / Assign</target>
         <note>key: role.policy.state.assign</note>
       </trans-unit>
+      <trans-unit id="cc6e4acc0dcae1afe8db5b49f7dd5edaae630335" resname="role.policy.tags">
```

Comment:
It was auto-generated, do we need this? ping @ViniTou @adamwojs @webhdx @Nattfarinn

Reply:
No, that's part of netgen tags.

Reply:
OK, I will delete this tomorrow morning.